### PR TITLE
add logs on browser selection logic

### PR DESCRIPTION
**Pull request by rpdome**
Intune/COBO is running into an issue where broker is not able to launch Chrome in their flow.

Adding logs to get more information on those devices.

### CODE REVIEW

File: common/src/main/java/com/microsoft/identity/common/internal/ui/browser/BrowserSelector.java [hunk 1]
```diff
@@ -82,31 +82,34 @@ public static Browser select(final Context context, final List<BrowserDescriptor
...
-            return false;
-        }
+        final String methodTag = TAG + ":matches";
 
         if (!StringUtil.equalsIgnoreCase(descriptor.getPackageName(), browser.getPackageName())) {
             return false;
         }
 
         if (!descriptor.getSignatureHashes().equals(browser.getSignatureHashes())) {
+            Logger.info(methodTag, "Signature hash does not match. " +
```
Comment (rodomsft) [inline common/src/main/java/com/microsoft/identity/common/internal/ui/browser/BrowserSelector.java:94]: Can you add the packageName in these? Or is it safe to assume there's just one safe package for now.

File: common/src/main/java/com/microsoft/identity/common/internal/ui/browser/BrowserSelector.java [hunk 2]
```diff
@@ -62,7 +62,7 @@ public class BrowserSelector {
     public static Browser select(final Context context, final List<BrowserDescriptor> browserSafeList) throws ClientException {
         final String methodTag = TAG + ":select";
         final List<Browser> allBrowsers = getAllBrowsers(context);
-        Logger.verbose(methodTag, "Select the browser to launch.");
+        Logger.info(methodTag, "Select the browser to launch from safeList size=" + browserSafeList.size());
```
Comment (rodomsft) [inline common/src/main/java/com/microsoft/identity/common/internal/ui/browser/BrowserSelector.java:65]: Log the size of allBrowsers as well. I'd prefer you log the actual package names but if that's a privacy concern I understand.

Comment (rpdome) [reply common/src/main/java/com/microsoft/identity/common/internal/ui/browser/BrowserSelector.java:65]: Not a privacy concern - i just didn't want to flush the logs. 

We definitely can do that in the private test build though.

**Comment by rpdome** [comment]
log added as part of https://github.com/AzureAD/microsoft-authentication-library-common-for-android/pull/1957

**Pull request closed without merging**